README.md

```diff
@@ -545,6 +546,147 @@ module.export = function migrateData(app) {
 }
 ```
 
+# Force Id
+
+Learn more about LoopBack's [forceId](https://loopback.io/doc/en/lb3/Model-definition-JSON-file.html).
+
+Every document stored in Cloudant has a unique id, stored as `_id`, and the database has built-in index for it. Retrieving data by its `_id` gives a better performance than querying by other field.
+
+If you have two models that allow customized `_id` as primary key set with ```"forceId": false``` in the model definition, it will result in a document update conflict.
```

Comment:
We recommend user to be careful when creating customized model `id` instead of using the auto-generated one.